test/life.c

```diff
@@ -3,13 +3,12 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
- 
+
 #define for_x for (int x = 0; x < w; x++)
 #define for_y for (int y = 0; y < h; y++)
```

Comment:
This is external code, see line 1, but I guess we never expect/want updates as it's just a test, so lgtm

Reply:
Will revert this one